### PR TITLE
hardware, nRF9160

### PR DESCRIPTION
**Pull request by MarkoPura**
Fixed Installing the NRF Connect SDK Windows home directory path, added Install West section for Windows

### CODE REVIEW

File: docs/partials/setup-west-nrf91-windows.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    :: PowerShell
+    golioth-ncs-workspace\.venv\Scripts\Activate.ps1
+
+    ```
+
+Once activated your shell will be prefixed with `(.venv)`. The virtual environment can be deactivated at any time by running `deactivate`.
+
+:::note
+Remember to activate the virtual environment every time you start working.
+:::
```
Comment (szczys) [inline docs/partials/setup-west-nrf91-windows.md:68]:
```suggestion
    Once activated your shell will be prefixed with `(.venv)`. The virtual environment can be deactivated at any time by running `deactivate`.

    :::tip Reactive for each new session
    Remember to activate the virtual environment every time you start working.
    :::

```

File: docs/partials/setup-west-nrf91-windows.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    pip3 install -U west
+    ```
+
+</TabItem>
+</Tabs>
+
+
+:::note
+
+Using the VS Code extension in conjuction with the nRF Connect for Desktop tools may move you outside many of the other recommended methods of compiling your firmware, described on this docs page and elsewhere on Golioth. If you're having problems with your Windows install, please contact us on our [Community Discord](https://golioth.io/discord)
```
Comment (szczys) [inline docs/partials/setup-west-nrf91-windows.md:90]:
```suggestion
Using the VS Code extension in conjuction with the nRF Connect for Desktop tools may move you outside many of the other recommended methods of compiling your firmware, described on this docs page and elsewhere on Golioth. If you're having problems with your Windows install, please contact us on [the Golioth Forum](https://forum.golioth.io).
```

File: docs/partials/setup-west-nrf91-windows.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    ```
+
+Once activated your shell will be prefixed with `(.venv)`. The virtual environment can be deactivated at any time by running `deactivate`.
+
+:::note
+Remember to activate the virtual environment every time you start working.
+:::
+
+3. Install west:
+    ```
```
Comment (szczys) [inline docs/partials/setup-west-nrf91-windows.md:71]:
```suggestion

    ```shell
```

File: docs/partials/setup-west-nrf91-windows.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    pip install west
+    ```
+
+</TabItem>
+<TabItem value="global">
+
+1. Install west:
+
+    ```
```
Comment (szczys) [inline docs/partials/setup-west-nrf91-windows.md:80]:
```suggestion
    ```shell
```

**Review by szczys — COMMENTED** [review]
This looks great @MarkoPura! I made some suggestions for you to review.

The one thing I noted in our call earlier today is that there was one line that had unicode open/close quotes where I think we should use ASCII. I mention it here because I changed those to tick marks but in other cases I would have changed to ASCII double quotes:

```
1. Open an Administrator `cmd.exe` window: press the Windows key, type “cmd.exe”, right-click the result, and choose “Run as Administrator”.
```

**Comment by MarkoPura** [comment]
@szczys, @ChrisGammell I've made some structural changes that made sense to me. Check them out and let me know if there's anything I should change.

**Comment by MarkoPura** [comment]
Install west section for Windows tested on a Windows 10 machine. West is installed by copying commands from the docs.

**Comment by szczys** [comment]
@MarkoPura is the change to where partials located related to the windows  install instructions? Can we separate those location changes into a different PR?

**Review by szczys — APPROVED** [review]
LGTM!

**Pull request merged**